### PR TITLE
Update website and documentation for version 0.4.0

### PR DESCRIPTION
**Pull request by PeterTh**
This should be merged once distributed scheduling is in, before the 0.4 release.

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -6,49 +6,42 @@ sidebar_label: Overview
...
-  graph would of course increase in size with additional nodes.
-- While in the animation workers proceed seemingly sequentially over data send
-  and receive operations, all of these are in fact processed asynchronously
-  in the actual implementation.
-
-Despite these simplifications, the animation captures the basic execution
+includes individual instructions for the given node in the system, and also
+encodes all necessary data transfers to maintain a consistent view of data
+as it would be expected if the program were to be executed on a single node.  
+In the Celerity **distributed scheduling** scheme, only the subset of the command graph necessary for executing locally relevant
```
Comment (psalz) [inline docs/overview.md:27]: "(introduced with Celerity 0.4.0)"

File: docs/overview.md [hunk 2]
```diff
@@ -6,49 +6,42 @@ sidebar_label: Overview
...
+consumed by __executor threads__, which finally leads to actually running the
+computations described by each kernel in the input program, distributed among 
+the GPUs in the cluster.
+
+In order to make the entire process more clear, this illustration is simplified
+in several aspects:
+
+- The details of data allocations, exchange and execution are omitted.
+- For this illustration, there are only three nodes in the system.
+- In practice, Horizons constrain the tasks and commands being actively processed
```
Comment (psalz) [inline docs/overview.md:41]: Link to horizons paper

File: docs/overview.md [hunk 3]
```diff
@@ -4,51 +4,43 @@ title: Overview
...
-Despite these simplifications, the animation captures the basic execution
+includes individual instructions for the given node in the system, and also
+encodes all necessary data transfers to maintain a consistent view of data
+as it would be expected if the program were to be executed on a single node.  
+In the Celerity **distributed scheduling** scheme (introduced with Celerity 0.4.0),
+only the subset of the command graph necessary for executing locally relevant
+commands is created on each node, so e.g. if nodes 0 and 1 communicate with each
+other locally, node 2 need not be aware of the related commands.
+
+On each node the jobs encoded withing the command graph are asynchronously
```
Comment (psalz) [inline docs/overview.md:30]:
```suggestion
On each node the jobs encoded within the command graph are asynchronously
```

File: docs/installation.md [hunk 4]
```diff
@@ -33,14 +33,6 @@ platforms.
 > hipSYCL is currently available on Linux and has experimental/partial support
 > for OSX and Windows.
 
-### ComputeCpp
```
Comment (psalz) [inline docs/installation.md:36]: We also still have this in our main README

File: website/pages/en/index.js [hunk 5]
```diff
@@ -103,22 +103,50 @@ class Index extends React.Component {
       </Block>
     );
 
-    const Disclaimer = () => (
```
Comment (psalz) [inline website/pages/en/index.js:106]: We should remove this from our main README as well

File: docs/installation.md (hunk 4 above)
Comment (fknorr) [reply docs/installation.md:36]: Removed.

File: website/pages/en/index.js (hunk 5 above)
Comment (fknorr) [reply website/pages/en/index.js:106]: Done.

File: docs/overview.md (hunk 3 above)
Comment (PeterTh) [reply docs/overview.md]: Done.

**Pull request merged**